converter.py

```diff
@@ -2,7 +2,7 @@
 import csv
 
 def convert():
-    with open("20201103__ri__general__precinct.csv", "wt") as csvfile:
+    with open("20201103__ri__primary__precinct.csv", "wt") as csvfile:
```

Comment:
Just need to fix the date segment on the file name to 20200908

Reply:
Good catch. Fixed.